app/js/services/DiseaseService.js

```diff
@@ -26,67 +42,68 @@ function DiseaseService($q, $resource, AppSettings, DiseaseModel, $log, filterFi
    *
```

Comment:
Line 37 of this comment above is no longer true, right? `DiseaseModels` come back as stubs, which can be "filled in" with a call to `.getAggregates()`.